api/libretime_api/storage/management/commands/bulk_import.py

```diff
@@ -52,7 +53,9 @@ def add_arguments(self, parser: CommandParser):
         )
 
     def handle(self, *args, **options):
-        url = settings.CONFIG.general.public_url
+        url = os.environ.get(
+            "LIBRETIME_GENERAL_PUBLIC_URL", settings.CONFIG.general.public_url
+        )
```

Comment:
This is not needed, the config already handle env vars by itself.

Reply:
in my testing bulk import tries to call `http://localhost:8000/api` etc. but the docker container can't actually access that. It needs to use http://nginx:8080 instead to access the API to run the import.

Reply:
`settings.CONFIG.general.public_url = http://localhost:8000/` is the default when you spin up the compose environment.